app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,19 @@
+from .dwarf import Dwarf
```

Comment:
relative imports is not a recommended way since imports may or may not work based on the directory from which python script is being run, please change this in all files